### ISSUE TITLE
BHT thermostat switch state

### ISSUE THREAD

**Issue by kizio1000** [make-all/tuya-local#140]
Hi,

Is there a way to create entity with current state of relay for family of BHT-like thermostats (BHT-002 etc.) ? Presently there is limited functionality as automation creation based on relay state is not possible.

**Comment by make-all** [comment]
I don't have one of these devices myself, so I cannot debug myself. There are a couple of unknown dps ids, in particular dp 104 is an unknown boolean that exists in BHT6000 and both variants of BHT002, which could be a candidate for such a state.

**Comment by make-all** [comment]
As no further information was provided in over 3 months, I will close this issue.  If you have any technical information that would help with implementing this, feel free to reopen it.

**Comment by cabe1214** [comment]
@make-all I am facing the same problem (not being able to determine if the Thermostat' status is "heating" or "standby". I have several of these devices and I'd be happy to support the debugging with some guidance.
Please feel free to reach out and I'll try to get the info you need. 

Thank you in advance!

**Comment by make-all** [comment]
If you can find the info that would inform which state it is in, then please reopen this with the info.

**Comment by cabe1214** [comment]
@make-all where can I get all the Info that the device is exposing?

**Comment by make-all** [comment]
See https://github.com/make-all/tuya-local#device-support

**Comment by cabe1214** [comment]
Unfortunately, I can confirm that the device is not exposing the information about the status of the internal relay (closed/open) and therefore we cannot determine the Heating/Standby state of the Thermostat.

I found some additional information about the topic [here](https://community.home-assistant.io/t/moes-bht-002-thermostat-local-control-tuya-based/151953/37).

TL;DR
You'd need to modify the hardware (soldering two resistors to some GPIO pins of the chip) for being able to "see" the switch status. On top of that, you'd also need to flash a custom firmware to expose that information to the outside.
